Add Contributing documentation

AutomatedTester opened the pull request:
When Pull requests and issues are opened, Github automatically links to
the contributing file so that people are aware of it. This hopefully
means that contributors will follow some of the rules.

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/mozilla/geckodriver/368)
<!-- Reviewable:end -->

jgraham commented:
Review status: 0 of 1 files reviewed at latest revision, 8 unresolved discussions, some commit checks failed.

---

*[CONTRIBUTING.md, line 15 at r1](https://reviewable.io:443/reviews/mozilla/geckodriver/368#-KXkkzhSTmhoP4_7HyJ6:-KXkkzhTRyoXA44hqldD:bxnqsqi) ([raw file](https://github.com/mozilla/geckodriver/blob/f3548fd11159be3d847ec7ba67c6acd53c17aac0/CONTRIBUTING.md#L15)):*
For many of the problems we get it's at least as useful to see the logs so it's clear what's being sent over the wire. Those are also easier to produce than a good minimal testcase. I think we should rewrite this section to be more geckodriver-specific e.g.:

> For issue reports to be actionable, it must be clear exactly what the observed and expected behaviours are, and how to set up the state required to observe the erroneous behaviour. The most useful thing to provide is a minimal HTML file which allows the problem to be reproduced, plus a debug-level log  from geckodriver showing the wire-protocol calls used to set up the problem. The  HTML file should ideally have only the code required to reproduce the problem with as little extraneous material as possible (it should not, for example, be a copy of your production site unless every element on the page is required to reproduce the bug). Because of the wide variety of client bindings for WebDriver, clients scripts and logs are typically not very useful if the verbose geckodriver logs are available. Issues relating to a specific client should be filed in the issue tracker of that project.

---

*[CONTRIBUTING.md, line 46 at r1](https://reviewable.io:443/reviews/mozilla/geckodriver/368#-KXlU7JqNg1wkQbxBUuZ:-KXlU7JrMy8P5-EligNX:bg87j5c) ([raw file](https://github.com/mozilla/geckodriver/blob/f3548fd11159be3d847ec7ba67c6acd53c17aac0/CONTRIBUTING.md#L46)):*
> ```Markdown
> ```
> 
> We practice HEAD-based development, which means all changes are applied
> ```

I can't tell what that actually means to judge if it's true or not.

---

*[CONTRIBUTING.md, line 62 at r1](https://reviewable.io:443/reviews/mozilla/geckodriver/368#-KXlUDx-pOXV2ucRPo4C:-KXlUDx0MtYRQ-UwFYgI:bwds59r) ([raw file](https://github.com/mozilla/geckodriver/blob/f3548fd11159be3d847ec7ba67c6acd53c17aac0/CONTRIBUTING.md#L62)):*
> ```Markdown
> any).  Follow these guidelines when writing one:
> 
> 1. The first line should be around 50 characters or less and contain a
> ```

That is not a "rule" that I'm interested in enforcing.

---

*[CONTRIBUTING.md, line 65 at r1](https://reviewable.io:443/reviews/mozilla/geckodriver/368#-KXlUOIqWlfLlXN2pw3Y:-KXlUOIqWlfLlXN2pw3Z:bny3he7) ([raw file](https://github.com/mozilla/geckodriver/blob/f3548fd11159be3d847ec7ba67c6acd53c17aac0/CONTRIBUTING.md#L65)):*
> ```Markdown
>    short description of the change.
> 2. Keep the second line blank.
> 3. Wrap all other lines at 72 columns.
> ```

Or that, really. Anything close to 80 columns seems fine. In general I don't think we should have a wall of rules to put off contributors, and I certainly wouldn't reject a patch for having 73 columns in the commit message (I might if it had like 200, so I think a guideline of "around 80" is fine).

---

*[CONTRIBUTING.md, line 69 at r1](https://reviewable.io:443/reviews/mozilla/geckodriver/368#-KXlUr1_nmpib1cYsRwJ:-KXlUr1_nmpib1cYsRwK:b-8anqgx) ([raw file](https://github.com/mozilla/geckodriver/blob/f3548fd11159be3d847ec7ba67c6acd53c17aac0/CONTRIBUTING.md#L69)):*
> ```Markdown
>    fixes, if any.
> 
> A good commit message can look like this:
> ```

I feel like this is a lot of information about trivia vs the size of the document. Can we remove this example?

---

*[CONTRIBUTING.md, line 109 at r1](https://reviewable.io:443/reviews/mozilla/geckodriver/368#-KXlV3RRuBc8mseAWKVh:-KXlV3RRuBc8mseAWKVi:bbfvvtb) ([raw file](https://github.com/mozilla/geckodriver/blob/f3548fd11159be3d847ec7ba67c6acd53c17aac0/CONTRIBUTING.md#L109)):*
> ```Markdown
> Pull requests are usually reviewed within a few days. If there are
> comments to address, apply your changes in new commits (preferably
> [fixups](http://git-scm.com/docs/git-commit)) and push to the same
> ```

I don't think we really care about fixups. Given that we use reviewable, amends are also OK. Mentioning that review will typically be conducted using reviewable seems like a more useful thing to mention here.

---

*[CONTRIBUTING.md, line 115 at r1](https://reviewable.io:443/reviews/mozilla/geckodriver/368#-KXlVJNpr8XVyHvcgLTR:-KXlVJNpr8XVyHvcgLTS:b-tjofkm) ([raw file](https://github.com/mozilla/geckodriver/blob/f3548fd11159be3d847ec7ba67c6acd53c17aac0/CONTRIBUTING.md#L115)):*
> ```Markdown
> 
> When code review is complete, a committer will take your PR and
> integrate it on Selenium's master branch. Because we like to keep a
> ```

This is not Selenium.

---

*[CONTRIBUTING.md, line 122 at r1](https://reviewable.io:443/reviews/mozilla/geckodriver/368#-KXlVNgddEETnTRlt5Cs:-KXlVNgddEETnTRlt5Ct:b-mzy6qq) ([raw file](https://github.com/mozilla/geckodriver/blob/f3548fd11159be3d847ec7ba67c6acd53c17aac0/CONTRIBUTING.md#L122)):*
> ```Markdown
> 
> GeckoDriver contributors frequent the `#ateam` channel on
> [`irc.freenode.org`](https://webchat.freenode.net/).
> ```

That would be `irc.mozilla.org`.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/mozilla/geckodriver/368)*
<!-- Sent from Reviewable.io -->

AutomatedTester commented:
Review status: 0 of 1 files reviewed at latest revision, 8 unresolved discussions, some commit checks failed.

---

*[CONTRIBUTING.md, line 15 at r1](https://reviewable.io:443/reviews/mozilla/geckodriver/368#-KXkkzhSTmhoP4_7HyJ6:-KXlqVKz338slk8HFWBV:b-7ql1td) ([raw file](https://github.com/mozilla/geckodriver/blob/f3548fd11159be3d847ec7ba67c6acd53c17aac0/CONTRIBUTING.md#L15)):*
<details><summary><i>Previously, jgraham wrote…</i></summary><blockquote>

For many of the problems we get it's at least as useful to see the logs so it's clear what's being sent over the wire. Those are also easier to produce than a good minimal testcase. I think we should rewrite this section to be more geckodriver-specific e.g.:

> For issue reports to be actionable, it must be clear exactly what the observed and expected behaviours are, and how to set up the state required to observe the erroneous behaviour. The most useful thing to provide is a minimal HTML file which allows the problem to be reproduced, plus a debug-level log  from geckodriver showing the wire-protocol calls used to set up the problem. The  HTML file should ideally have only the code required to reproduce the problem with as little extraneous material as possible (it should not, for example, be a copy of your production site unless every element on the page is required to reproduce the bug). Because of the wide variety of client bindings for WebDriver, clients scripts and logs are typically not very useful if the verbose geckodriver logs are available. Issues relating to a specific client should be filed in the issue tracker of that project.
</blockquote></details>

fixed

---

*[CONTRIBUTING.md, line 46 at r1](https://reviewable.io:443/reviews/mozilla/geckodriver/368#-KXlU7JqNg1wkQbxBUuZ:-KXlnfscu0-e_nY-3Qkc:b-qdl97n) ([raw file](https://github.com/mozilla/geckodriver/blob/f3548fd11159be3d847ec7ba67c6acd53c17aac0/CONTRIBUTING.md#L46)):*
<details><summary><i>Previously, jgraham wrote…</i></summary><blockquote>

I can't tell what that actually means to judge if it's true or not.
</blockquote></details>

It's true :)

---

*[CONTRIBUTING.md, line 62 at r1](https://reviewable.io:443/reviews/mozilla/geckodriver/368#-KXlUDx-pOXV2ucRPo4C:-KXlqWxmJPAWngiicE9p:bvb87mw) ([raw file](https://github.com/mozilla/geckodriver/blob/f3548fd11159be3d847ec7ba67c6acd53c17aac0/CONTRIBUTING.md#L62)):*
<details><summary><i>Previously, jgraham wrote…</i></summary><blockquote>

That is not a "rule" that I'm interested in enforcing.
</blockquote></details>

removed this section

---

*[CONTRIBUTING.md, line 65 at r1](https://reviewable.io:443/reviews/mozilla/geckodriver/368#-KXlUOIqWlfLlXN2pw3Y:-KXlnrc1EPx6VUJCA5_1:b-sfa1qy) ([raw file](https://github.com/mozilla/geckodriver/blob/f3548fd11159be3d847ec7ba67c6acd53c17aac0/CONTRIBUTING.md#L65)):*
<details><summary><i>Previously, jgraham wrote…</i></summary><blockquote>

Or that, really. Anything close to 80 columns seems fine. In general I don't think we should have a wall of rules to put off contributors, and I certainly wouldn't reject a patch for having 73 columns in the commit message (I might if it had like 200, so I think a guideline of "around 80" is fine).
</blockquote></details>

I'll remove this section and we can guide people if they have rubbish commits

---

*[CONTRIBUTING.md, line 69 at r1](https://reviewable.io:443/reviews/mozilla/geckodriver/368#-KXlUr1_nmpib1cYsRwJ:-KXlqZ8C4gzWVGDXUcAS:bvb87mw) ([raw file](https://github.com/mozilla/geckodriver/blob/f3548fd11159be3d847ec7ba67c6acd53c17aac0/CONTRIBUTING.md#L69)):*
<details><summary><i>Previously, jgraham wrote…</i></summary><blockquote>

I feel like this is a lot of information about trivia vs the size of the document. Can we remove this example?
</blockquote></details>

removed this section

---

*[CONTRIBUTING.md, line 109 at r1](https://reviewable.io:443/reviews/mozilla/geckodriver/368#-KXlV3RRuBc8mseAWKVh:-KXlqRAcR32hfw7hNUQN:b-896fix) ([raw file](https://github.com/mozilla/geckodriver/blob/f3548fd11159be3d847ec7ba67c6acd53c17aac0/CONTRIBUTING.md#L109)):*
<details><summary><i>Previously, jgraham wrote…</i></summary><blockquote>

I don't think we really care about fixups. Given that we use reviewable, amends are also OK. Mentioning that review will typically be conducted using reviewable seems like a more useful thing to mention here.
</blockquote></details>

Done.

---

*[CONTRIBUTING.md, line 115 at r1](https://reviewable.io:443/reviews/mozilla/geckodriver/368#-KXlVJNpr8XVyHvcgLTR:-KXloHjgSG9epBBi7TSY:b-896fix) ([raw file](https://github.com/mozilla/geckodriver/blob/f3548fd11159be3d847ec7ba67c6acd53c17aac0/CONTRIBUTING.md#L115)):*
<details><summary><i>Previously, jgraham wrote…</i></summary><blockquote>

This is not Selenium.
</blockquote></details>

Done.

---

*[CONTRIBUTING.md, line 122 at r1](https://reviewable.io:443/reviews/mozilla/geckodriver/368#-KXlVNgddEETnTRlt5Cs:-KXlqPTnmrUbe9zdyJ0B:b-896fix) ([raw file](https://github.com/mozilla/geckodriver/blob/f3548fd11159be3d847ec7ba67c6acd53c17aac0/CONTRIBUTING.md#L122)):*
<details><summary><i>Previously, jgraham wrote…</i></summary><blockquote>

That would be `irc.mozilla.org`.
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/mozilla/geckodriver/368)*
<!-- Sent from Reviewable.io -->

jgraham commented:
Review status: 0 of 1 files reviewed at latest revision, 3 unresolved discussions, some commit checks failed.

---

*[CONTRIBUTING.md, line 46 at r1](https://reviewable.io:443/reviews/mozilla/geckodriver/368#-KXlU7JqNg1wkQbxBUuZ:-KXqN4UDLv8-3LBj9362:b2s2u1w) ([raw file](https://github.com/mozilla/geckodriver/blob/f3548fd11159be3d847ec7ba67c6acd53c17aac0/CONTRIBUTING.md#L46)):*
<details><summary><i>Previously, AutomatedTester (David Burns) wrote…</i></summary><blockquote>

It's true :)
</blockquote></details>

OK, but since I don't know what it means I think it's probably OK to remove from this document.

---

*[CONTRIBUTING.md, line 10 at r2](https://reviewable.io:443/reviews/mozilla/geckodriver/368#-KXqNLA1spgQl_fvs4UW:-KXqNLA1spgQl_fvs4UX:b-pvea8k) ([raw file](https://github.com/mozilla/geckodriver/blob/56fdfc32273cbc2d0176b26b3f2582a35c0d0bf7/CONTRIBUTING.md#L10)):*
> GeckoDriver

I think we spell this as `geckodriver` in other places (change here and throughout).

---

*[CONTRIBUTING.md, line 19 at r2](https://reviewable.io:443/reviews/mozilla/geckodriver/368#-KXqMbG6A9fDfkIOUMaE:-KXqMbG6A9fDfkIOUMaF:b-1kqibv) ([raw file](https://github.com/mozilla/geckodriver/blob/56fdfc32273cbc2d0176b26b3f2582a35c0d0bf7/CONTRIBUTING.md#L19)):*
> ```Markdown
> up the problem. Please provide [concise reproducible test
> cases](http://sscce.org/) and describe what results you are seeing and what
> results you expect.. Because of the wide variety of client bindings for
> ```

Two full stops.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/mozilla/geckodriver/368)*
<!-- Sent from Reviewable.io -->

jgraham commented:
Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 2 unresolved discussions, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/mozilla/geckodriver/368)*
<!-- Sent from Reviewable.io -->

andreastt commented on CONTRIBUTING.md:
````diff
@@ -0,0 +1,99 @@
+### Step 5: Push
+
+```text
+% git push origin my-feature-branch
````
`origin` is superfluous here.

andreastt commented on CONTRIBUTING.md:
````diff
@@ -0,0 +1,99 @@
+% git push origin my-feature-branch
+```
+
+Go to https://github.com/yourusername/geckodriver.git and press the _Pull
````
s/\.git//

andreastt commented on CONTRIBUTING.md:
```diff
@@ -0,0 +1,99 @@
+Request_ and fill out the form.
+
+Pull requests are usually reviewed within a few days. Reviews will be done
+through [Reviewable](https://reviewable.io/reviews/mozilla/geckodriver)
```
FWIW I’m unable to log in to Reviewable right now and I frequently do reviews on GitHub. Perhaps just leave it out?

andreastt commented on CONTRIBUTING.md:
```diff
@@ -0,0 +1,99 @@
+# Contributing to GeckoDriver
```
s/GeckoDriver/geckodriver/g

andreastt commented on CONTRIBUTING.md:
```diff
@@ -0,0 +1,99 @@
+The GeckoDriver project welcomes contributions from everyone. There are a
+number of ways you can help:
+
+## Bug Reports
```
s/Bug/Issue/

andreastt commented on CONTRIBUTING.md:
```diff
@@ -0,0 +1,99 @@
+observed and expected behaviours are, and how to set up the state required
+to observe the erroneous behaviour. The most useful thing to provide is a
+minimal HTML file which allows the problem to be reproduced, plus a
+debug-level log from geckodriver showing the wire-protocol calls used to set
```
s/debug/trace/

andreastt commented on CONTRIBUTING.md:
```diff
@@ -0,0 +1,99 @@
+debug-level log from geckodriver showing the wire-protocol calls used to set
+up the problem. Please provide [concise reproducible test
+cases](http://sscce.org/) and describe what results you are seeing and what
+results you expect.. Because of the wide variety of client bindings for
```
Two full stops.

andreastt commented on CONTRIBUTING.md:
```diff
@@ -0,0 +1,99 @@
+
+## Code Contributions
+
+The GeckoDriver project welcomes new contributors.
```
We already say this above.

andreastt commented on CONTRIBUTING.md:
```diff
@@ -0,0 +1,99 @@
+
+If you're looking for easy bugs, have a look at
+[issues labelled E-easy](https://github.com/mozilla/geckodriver/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aissue+label%3Aeasy+)
+on Github.
```
s/Github/GitHub/g apparently.

andreastt commented on CONTRIBUTING.md:
```diff
@@ -0,0 +1,99 @@
+## Communication
+
+GeckoDriver contributors frequent the `#ateam` channel on
+[`irc.mozilla.org`](http://chat.mibbit.com/?server=irc.mozilla.org:#ateam).
```
Point out that this is an IRC channel.

AutomatedTester commented:
Review status: 0 of 1 files reviewed at latest revision, 12 unresolved discussions, some commit checks failed.

---

*[CONTRIBUTING.md, line 1 at r2](https://reviewable.io:443/reviews/mozilla/geckodriver/368#-KXkcM6GEyKF87KVQhOq-r2-1:-KYVj0oRqdCK0L0coXB7:b-896fix) ([raw file](https://github.com/mozilla/geckodriver/blob/56fdfc32273cbc2d0176b26b3f2582a35c0d0bf7/CONTRIBUTING.md#L1)):*
<details><summary><i>Previously, andreastt (Andreas Tolfsen) wrote…</i></summary><blockquote>

s/GeckoDriver/geckodriver/g
</blockquote></details>

Done.

---

*[CONTRIBUTING.md, line 6 at r2](https://reviewable.io:443/reviews/mozilla/geckodriver/368#-KXkcM6GEyKF87KVQhOq-r2-6:-KYVj09NOOdySXLMkRNk:b-896fix) ([raw file](https://github.com/mozilla/geckodriver/blob/56fdfc32273cbc2d0176b26b3f2582a35c0d0bf7/CONTRIBUTING.md#L6)):*
<details><summary><i>Previously, andreastt (Andreas Tolfsen) wrote…</i></summary><blockquote>

s/Bug/Issue/
</blockquote></details>

Done.

---

*[CONTRIBUTING.md, line 10 at r2](https://reviewable.io:443/reviews/mozilla/geckodriver/368#-KXqNLA1spgQl_fvs4UW:-KYVj-XNF_0IZWhf1UEX:b-896fix) ([raw file](https://github.com/mozilla/geckodriver/blob/56fdfc32273cbc2d0176b26b3f2582a35c0d0bf7/CONTRIBUTING.md#L10)):*
<details><summary><i>Previously, jgraham wrote…</i></summary><blockquote>

> GeckoDriver

I think we spell this as `geckodriver` in other places (change here and throughout).

</blockquote></details>

Done.

---

*[CONTRIBUTING.md, line 16 at r2](https://reviewable.io:443/reviews/mozilla/geckodriver/368#-KXkcM6GEyKF87KVQhOq-r2-16:-KYVizoURr5xCDm0XyVs:b-896fix) ([raw file](https://github.com/mozilla/geckodriver/blob/56fdfc32273cbc2d0176b26b3f2582a35c0d0bf7/CONTRIBUTING.md#L16)):*
<details><summary><i>Previously, andreastt (Andreas Tolfsen) wrote…</i></summary><blockquote>

s/debug/trace/
</blockquote></details>

Done.

---

*[CONTRIBUTING.md, line 19 at r2](https://reviewable.io:443/reviews/mozilla/geckodriver/368#-KXkcM6GEyKF87KVQhOq-r2-19:-KYViyGQ_GetbOFXDbe8:b-896fix) ([raw file](https://github.com/mozilla/geckodriver/blob/56fdfc32273cbc2d0176b26b3f2582a35c0d0bf7/CONTRIBUTING.md#L19)):*
<details><summary><i>Previously, andreastt (Andreas Tolfsen) wrote…</i></summary><blockquote>

Two full stops. 
</blockquote></details>

Done.

---

*[CONTRIBUTING.md, line 19 at r2](https://reviewable.io:443/reviews/mozilla/geckodriver/368#-KXqMbG6A9fDfkIOUMaE:-KYViyvS4IezbqK0VAXv:b-896fix) ([raw file](https://github.com/mozilla/geckodriver/blob/56fdfc32273cbc2d0176b26b3f2582a35c0d0bf7/CONTRIBUTING.md#L19)):*
<details><summary><i>Previously, jgraham wrote…</i></summary><blockquote>

Two full stops.
</blockquote></details>

Done.

---

*[CONTRIBUTING.md, line 26 at r2](https://reviewable.io:443/reviews/mozilla/geckodriver/368#-KXkcM6GEyKF87KVQhOq-r2-26:-KYVixcAq3cRj2qKX-F-:b-896fix) ([raw file](https://github.com/mozilla/geckodriver/blob/56fdfc32273cbc2d0176b26b3f2582a35c0d0bf7/CONTRIBUTING.md#L26)):*
<details><summary><i>Previously, andreastt (Andreas Tolfsen) wrote…</i></summary><blockquote>

We already say this above.
</blockquote></details>

Done.

---

*[CONTRIBUTING.md, line 30 at r2](https://reviewable.io:443/reviews/mozilla/geckodriver/368#-KXkcM6GEyKF87KVQhOq-r2-30:-KYViwxJP1OGPAWtXBLK:b-896fix) ([raw file](https://github.com/mozilla/geckodriver/blob/56fdfc32273cbc2d0176b26b3f2582a35c0d0bf7/CONTRIBUTING.md#L30)):*
<details><summary><i>Previously, andreastt (Andreas Tolfsen) wrote…</i></summary><blockquote>

s/Github/GitHub/g apparently.
</blockquote></details>

Done.

---

*[CONTRIBUTING.md, line 80 at r2](https://reviewable.io:443/reviews/mozilla/geckodriver/368#-KXkcM6GEyKF87KVQhOq-r2-80:-KYViktrY_0_7sZ_O0X0:bnleb6z) ([raw file](https://github.com/mozilla/geckodriver/blob/56fdfc32273cbc2d0176b26b3f2582a35c0d0bf7/CONTRIBUTING.md#L80)):*
<details><summary><i>Previously, andreastt (Andreas Tolfsen) wrote…</i></summary><blockquote>

`origin` is superfluous here.
</blockquote></details>

I am going to leave it here.  A number of auto-complete tools only work if the remote name is used before the branch

---

*[CONTRIBUTING.md, line 83 at r2](https://reviewable.io:443/reviews/mozilla/geckodriver/368#-KXkcM6GEyKF87KVQhOq-r2-83:-KYVijvui-smEOA8LAKa:b-896fix) ([raw file](https://github.com/mozilla/geckodriver/blob/56fdfc32273cbc2d0176b26b3f2582a35c0d0bf7/CONTRIBUTING.md#L83)):*
<details><summary><i>Previously, andreastt (Andreas Tolfsen) wrote…</i></summary><blockquote>

s/\.git//
</blockquote></details>

Done.

---

*[CONTRIBUTING.md, line 87 at r2](https://reviewable.io:443/reviews/mozilla/geckodriver/368#-KXkcM6GEyKF87KVQhOq-r2-87:-KYVigMzShpV63uOqVEG:bon1ivh) ([raw file](https://github.com/mozilla/geckodriver/blob/56fdfc32273cbc2d0176b26b3f2582a35c0d0bf7/CONTRIBUTING.md#L87)):*
<details><summary><i>Previously, andreastt (Andreas Tolfsen) wrote…</i></summary><blockquote>

FWIW I’m unable to log in to Reviewable right now and I frequently do reviews on GitHub. Perhaps just leave it out?
</blockquote></details>

I am going to leave it. 

---

*[CONTRIBUTING.md, line 99 at r2](https://reviewable.io:443/reviews/mozilla/geckodriver/368#-KXkcM6GEyKF87KVQhOq-r2-99:-KYVG9nDiLthfYi73nQr:b-7s73yx) ([raw file](https://github.com/mozilla/geckodriver/blob/56fdfc32273cbc2d0176b26b3f2582a35c0d0bf7/CONTRIBUTING.md#L99)):*
<details><summary><i>Previously, andreastt (Andreas Tolfsen) wrote…</i></summary><blockquote>

Point out that this is an IRC channel.
</blockquote></details>

Adding the channel in the url just cuts out the step to pick a channel and the channel name is already mentioned in the channel

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/mozilla/geckodriver/368)*
<!-- Sent from Reviewable.io -->

andreastt commented:
Review status: 0 of 1 files reviewed at latest revision, 5 unresolved discussions, some commit checks failed.

---

*[CONTRIBUTING.md, line 80 at r2](https://reviewable.io:443/reviews/mozilla/geckodriver/368#-KXkcM6GEyKF87KVQhOq-r2-80:-KYWILAz6CJBVaVNA1O8:b32w0x) ([raw file](https://github.com/mozilla/geckodriver/blob/56fdfc32273cbc2d0176b26b3f2582a35c0d0bf7/CONTRIBUTING.md#L80)):*
<details><summary><i>Previously, AutomatedTester (David Burns) wrote…</i></summary><blockquote>

I am going to leave it here.  A number of auto-complete tools only work if the remote name is used before the branch
</blockquote></details>

OK

---

*[CONTRIBUTING.md, line 87 at r2](https://reviewable.io:443/reviews/mozilla/geckodriver/368#-KXkcM6GEyKF87KVQhOq-r2-87:-KYWIP5a5XUntx-y46tK:b32w0x) ([raw file](https://github.com/mozilla/geckodriver/blob/56fdfc32273cbc2d0176b26b3f2582a35c0d0bf7/CONTRIBUTING.md#L87)):*
<details><summary><i>Previously, AutomatedTester (David Burns) wrote…</i></summary><blockquote>

I am going to leave it. 
</blockquote></details>

OK

---

*[CONTRIBUTING.md, line 99 at r2](https://reviewable.io:443/reviews/mozilla/geckodriver/368#-KXkcM6GEyKF87KVQhOq-r2-99:-KYWISjNk1YBTN0sREC1:bvjd2do) ([raw file](https://github.com/mozilla/geckodriver/blob/56fdfc32273cbc2d0176b26b3f2582a35c0d0bf7/CONTRIBUTING.md#L99)):*
<details><summary><i>Previously, AutomatedTester (David Burns) wrote…</i></summary><blockquote>

Adding the channel in the url just cuts out the step to pick a channel and the channel name is already mentioned in the channel
</blockquote></details>

Right, yes. LGTM.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/mozilla/geckodriver/368)*
<!-- Sent from Reviewable.io -->

andreastt commented:
@AutomatedTester Since jgraham is on PTO, feel free to land this at your convenience. I believe most if not all the issues have been clarified.
